lib/middleware/defaultTopicTriggers/index/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
To keep consistent with the topic index route. Should the name of this middleware be `defaultTopicTriggers-get.js`?